### PR TITLE
Release RC with signed uploads

### PR DESCRIPTION
**Pull request by bautrukevich**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,14 +6,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+## [2.3.0-rc] - 2020-01-09
```
Comment (dmitry-mukhin) [inline CHANGELOG.md]:
```suggestion
## [2.4.0-rc] - 2020-01-09
```

**Comment by bautrukevich** [comment]
> we need to update lib version as well.

Yes, you're right

**Comment by bautrukevich** [comment]
@dmitry-mukhin waiting for your approval, please

**Comment by bautrukevich** [comment]
> we still need to change lib version here
> https://github.com/uploadcare/uploadcare-php/blob/master/src/Uploadcare/Api.php#L13

Fixed, thank you

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,14 +6,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+## [2.4.0-rc] - 2020-01-09
```
Comment (dmitry-mukhin) [inline CHANGELOG.md]:
```suggestion
## [2.4.0-rc] - 2020-01-13
```

**Pull request merged**